crds/defs.go

```diff
@@ -37,18 +35,17 @@ func (c crdFile) mustUnmarshal() *apiextensionsv1.CustomResourceDefinition {
 	reader := bytes.NewReader(data)
 	decoder := yaml.NewYAMLOrJSONDecoder(reader, 30)
 	if err = decoder.Decode(u); err != nil {
```

Comment:
Is it necessary to unmarshal twice? As far as I can tell, it's safe to decode directly into the v1 struct and sanity check its TypeMeta before trusting its other fields.

Reply:
That's a good point! Let's do that.